Verkiezingen_functies.py

```diff
@@ -115,27 +115,62 @@ def provincie_stemmen(provincie):   #deze functie pakt de lijst van gemeentes pe
     df_provincie2=df_provincie.iloc[:,0:47]
     return df_provincie2
 
-def provincie_als_landelijk():     #pakt de zetelverdeling op provincie niveau en voegt deze samen tot nieuwe dataframe, met op het einde de normale landelijke uitslag.
+def provincie_als_landelijk(inputDrenthe,inputNoord_Holland,inputGelderland,inputFriesland,Zuid_Holland,inputOverijssel,inputFlevoland,Noord_Brabant,inputUtrecht,inputGroningen,inputLimburg,inputZeeland):     #pakt de zetelverdeling op provincie niveau en voegt deze samen tot nieuwe dataframe, met op het einde de normale landelijke uitslag.
```

Comment:
Voor wat cleanere en intuïtievere code zou ik hier eerder een **kwargs input van maken. Dat betekent dat je met keywords optionele parameters mee kan geven, waarin je de weights dus zou kunnen aangeven. Kwargs werkt dan als een dict in je functie. Dan kun je in de functie een for loop maken waarin je door de provincies loopt, en checkt of er een weight in de kwargs zit, zo niet geef je een standaardwaarde.

Reply:
ik zou eerder denken aan *arg, we gebruiken de index/key van de series gewichten (provincie+'aantal zetels') zodat pandas daar niet over zeurt. maar hetzelfde als met *args kan als je in de endpoint gewoon zegt ( inputDrenthe= inputNoord_Holland = ... = inputZeeland=1 ) voor het geval dat er niks wordt ingevuld. verder moet de lijst altijd 12 zijn omdat we dingen met dot multiplicatie doen (matrix multiplicatie moet zelfde grote hebben op 1 van de 2 assen) dus vandaar dat een wat striktere aanwijzing imo beter is